src/utils/__tests__/track/index.spec.js

```diff
@@ -0,0 +1,108 @@
+/**
+Copyright 2016 Split Software
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+**/
+import tape from 'tape';
+import validate from '../../track/validate';
+
+tape('TRACK VALIDATE / if valid arguments are passed should return true', assert => {
+  const key = 'some_key';
```

Comment:
Just in the sake of not having more and more tests lines (we already have a ton) there's no need for creating a variable if you're not going to reuse it on an assert or something. 

Also you can kind of group the test cases for valid and invalid in two different test context, and vary the message that says: `track validate will return true if all params are valids`.

For example:
tape('TRACK VALIDATE / If valid arguments are passed it should always return true, assert => {
   assert.ok(areValidTrackArguments(...args1), 'Should be valid if everything is defined with valid types');
   assert.ok(areValidTrackArguments(...args2), 'Should be valid if everything is defined but the value is undefined');
  assert.ok(areValidTrackArguments(...args3), 'Should be valid if everything is defined but the value is null');
 ....
})